spec/features/initial_chatroom_spec.rb

```diff
@@ -18,4 +18,35 @@
       expect(page).to have_content("message stuff")
     end
   end
+
+  scenario "user sees last five messages when entering the main chat room", js: true do
+    messages = []
+    5.times do |n|
+      message = Message.create(body: "message number #{n}",
+                     channel: "main",
+                     user_id: user.id
+                    )
+      messages.push(message)
+    end
+
+    login_as(user)
+
+    messages.each do |message|
+      within('.messages') { expect(page).to have_content(message.body) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.